extensions/amp-consent/0.1/test/test-consent-info.js

```diff
@@ -75,19 +79,36 @@ describes.fakeWin('ConsentInfo', {}, () => {
           'consentState': CONSENT_ITEM_STATE.REJECTED,
           'consentString': 'test',
           'isDirty': undefined,
+          'consentMetadata': undefined,
+        })
+      );
+      expect(
+        getStoredConsentInfo({
+          's': 0,
+          'r': 'test',
+          'm': {},
+        })
+      ).to.deep.equal(
+        dict({
+          'consentState': CONSENT_ITEM_STATE.REJECTED,
+          'consentString': 'test',
+          'isDirty': undefined,
+          'consentMetadata': undefined,
         })
       );
       expect(
         getStoredConsentInfo({
           's': -1,
           'r': 'test',
           'd': 1,
+          'm': {},
```

Comment:
when would `m` be an empty object?

Reply:
Shouldn't happen, but I left it in there as a reminder to add tests here. I'll remove for now.